agent-any/src/AnyAgentComplex.cc

```diff
@@ -511,6 +517,13 @@ AnyAgent::parseData (char const *&line, const YCPValue & syntax, bool optional)
 		tupleName.push ("");
 		tupleValue.push (YCPNull ());
 		YCPValue tv = parseTuple (line, term->args (), optional);
+        if (tv.isNull())
+        {
+            if (optional)
+                y2milestone("Optional Tuple not found at line: '%s'", line);
+            else
+                y2error("Tuple not found at line: '%s'", line);
```

Comment:
in general seeing output include newline. I think it will be better to strip it ( in all logging of line )